.github/workflows/checks-vizro-ai.yml

```diff
@@ -42,9 +46,6 @@ jobs:
       - name: List dependencies
         run: hatch run all.py${{ matrix.python-version }}:pip freeze
 
-      - name: Lint
```

Comment:
Some parts here are still outdated, see this change + comment: https://github.com/mckinsey/vizro/pull/142/files#r1378559456

'hatch run docs:changelog' doesn't exist and has been updated - could you double-check?